sovrin_client/test/cli/test_command_reg_ex.py

```diff
@@ -43,6 +45,26 @@ def testSendNymWithVerkey(grammar):
     })
 
 
+def testGetNym(grammar):
+    dest = "LNAyBZUjvLF7duhrNtOWgdAKs18nHdbJUxJLT39iEGU="
+    matchedVars = getMatchedVariables(
+        grammar, "send GET_NYM dest={}".format(dest))
```

Comment:
Yes, the test will break if we rename the command.
But on the other hand I think it's OK to have hard-coded values in the tests.
It prevents from accidental renaming of the command. The CLI API is fixed to use 'send GET_NYM', and that's what we test here (hard-coded). If someone accidentally renames it, it may break CLI API. If we used a parameter for GET_NYM here, then it would not be noticed.
So, I would vote to use hard-coded values, if we test a public CLI API. It will not be a big deal to fix this test once we rename a command, but it guarantees that such a rename is expected.